app/views/records/edit_fields/_rights.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= f.input :rights, as: :select_with_modal_help,
+<%= f.input :rights, as: :select,
     collection: RightsService.select_options,
-    input_html: { class: 'form-control', multiple: true } %>
+    input_html: { class: 'form-control' } %>
```

Comment:
Not that I can recall, @cam156. There was a ticket for this, but we deferred it: https://github.com/projecthydra/sufia/issues/1287